src/cli-client/internals/cli/cli_upload.go

```diff
@@ -0,0 +1,150 @@
+package cli
+
+import (
+	"fmt"
+	"regexp"
+	"slices"
+	"strings"
+
+	"github.com/canonical/oci-factory/src/cli-client/internals/client"
+	"github.com/canonical/oci-factory/src/cli-client/internals/trigger"
+)
+
+type UploadRelease struct {
+	Track     string
+	Risks     []string
+	EndOfLife string
+}
+
+type CmdUpload struct {
+	UploadRelease []string `long:"release" description:"Release images to container registries.\nSyntax: --release track=<release track>,risks=<risk1>[,<risk2>...],eol=yyyy-mm-dd"`
```

Comment:
nit: How about `Release` as the struct field?

Reply:
Do you mean the `Release` from `package trigger`?

Reply:
No no, I meant renaming `UploadRelease` to `Release`.